src/client/app/component/CommentList/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './style.scss';
+import SingleComment from '../SingleComment';
+
+class CommentList extends React.Component {
+  render() {
+    let allComments = [];
+    const zero = 0;
+
+    if (this.props.comments && this.props.comments.length > zero) {
+      allComments = this.props.comments.map(
+        (element, index) => <SingleComment commentInfo={element}/>
```

Comment:
you don't use the index can be `element => <SingleComment commentInfo={element} />` also maybe you can rename the props to `info` since its on the `SingleComment` component